templates/education/education-side.njk

```diff
@@ -11,8 +20,8 @@
     <span class="glyphicon glyphicon-new-window"></span> View Map
 </a>
 <hr>
-{% include "partials/course-cancellation.njk" %}
+{# {% include "education/course-cancellation.njk" %} #}
```

Comment:
why does that file exist? this comments it out but the content is also in the markdown file which is where you need it.